src/hotspot/share/memory/allocation.hpp

```diff
@@ -283,10 +283,10 @@ class CHeapObj {
 
 class StackObj {
  private:
-  void* operator new(size_t size) throw();
-  void* operator new [](size_t size) throw();
-  void  operator delete(void* p);
-  void  operator delete [](void* p);
+  void* operator new(size_t size) throw() = delete;
+  void* operator new [](size_t size) throw() = delete;
+  void  operator delete(void* p) = delete;
+  void  operator delete [](void* p) = delete;
```

Comment:
Making these deleted functions public might provide better error messages if someone accidentally attempts to reference them.

Reply:
Done.